### PR TITLE
Add update_document script, implementing delete.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Tested manually by creating a fake SOLR record and calling delete.

```
ubuntu@production-console:~/production/bin/oneoff$ ./solr_test delete user 987654321
DELETING: user-987654321
```

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Early next week, I plan on adding `add` and `update`.

File: bin/solr/update_document [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  solr_id = nil
+  case type
+  when 'form'
+    solr_id = db_id
+  when 'user'
+    solr_id = "user-#{db_id}"
+  end
+
+  unless solr_id.nil?
+    puts "DELETING: #{solr_id}..."
```
Comment (tanyaparker) [inline bin/solr/update_document:29]: Is it worth it to put an error message if we didn't find a solr_id?

**Comment by tanyaparker** [comment]
Just one thought, but otherwise lgtm!

File: bin/solr/update_document (hunk 1 above)
Comment (ashercodeorg) [inline bin/solr/update_document:29]: Probably. For simplicity, going to leave that to another PR.
